deployability/modules/testing/testing.py

```diff
@@ -24,14 +29,17 @@ def run(cls, payload: InputPayload) -> None:
         cls._setup(ansible, extra_vars['working_dir'])
         cls._run_tests(payload.tests, ansible, extra_vars)
         if payload.cleanup:
+            logger.info("Cleaning up")
             cls._cleanup(ansible, extra_vars['working_dir'])
 
     @classmethod
     def _get_extra_vars(cls, payload: InputPayload) -> ExtraVars:
         if not payload.dependencies:
+            logger.debug("No dependencies found")
```

Comment:
If you add where I look for it, it improves the log.

Reply:
I think it could be improved by using
```suggestion
            logger.debug("No dependencies received in payload")
```